src/send/EnterAmount.test.tsx

```diff
@@ -320,6 +320,26 @@ describe('EnterAmount', () => {
       expect(tokenAmountInput.props.value).toBe(replaceSeparators('1000'))
       expect(localAmountInput.props.value).toBe(replaceSeparators('₱133'))
     })
+
+    it('entering MAX token applies correct decimal separator', async () => {
+      const store = createMockStore(mockStore)
+      const tokenBalances = mockStoreBalancesToTokenBalances([
+        { ...mockStoreTokenBalances[mockCeloTokenId], balance: '100000' },
```

Comment:
Can we include decimals in the balance? Otherwise I believe this test will pass even without the fix

Reply:
Fixed in 3bdc56680ff84de59208546f881e760f549d0551!